### PR TITLE
update using experiments section of the docs.

### PR DESCRIPTION
**Pull request by kkvtran**
## Description

Update documentation to include inference in the Using Experiment sections.

### CODE REVIEW

File: docs/source/using_experiments.rst [hunk 1]
```diff
@@ -4,18 +4,19 @@ Using Experiments
 Since version ``0.3``, the toolkit includes support for running ``Experiments``.
 An **Experiment** represents a high-level use case, such as training a neural
 network, in a compact form that allows for easily running the experiment and
-variations of it with ease both locally and remotely.
+variations of it with ease both locally and in the cloud.
```
Comment (kaoutar55) [inline docs/source/using_experiments.rst:7]: and its variations

File: docs/source/using_experiments.rst [hunk 2]
```diff
@@ -60,12 +61,14 @@ retrieving its results.
...
-========================================================  ========
-Tile class                                                Description
-========================================================  ========
-:class:`~aihwkit.experiments.runners.local.LocalRunner`   Runner for executing experiments locally
-:class:`~aihwkit.experiments.runners.cloud.CloudRunner`   Runner for executing experiments in the cloud
-========================================================  ========
+===================================================================  ========
+Tile class                                                           Description
+===================================================================  ========
+:class:`~aihwkit.experiments.runners.local.LocalRunner`              Runner for executing a training experiments locally
```
Comment (kaoutar55) [inline docs/source/using_experiments.rst:67]: Change: Runner for executing training experiments locally (remote a)

File: docs/source/using_experiments.rst [hunk 3]
```diff
@@ -60,12 +61,14 @@ retrieving its results.
...
+:class:`~aihwkit.experiments.runners.cloud.CloudRunner`              Runner for executing a training experiments in the cloud
```
Comment (kaoutar55) [inline docs/source/using_experiments.rst:68]: remove "a" before training

File: docs/source/using_experiments.rst [hunk 4]
```diff
@@ -60,12 +61,14 @@ retrieving its results.
...
+:class:`~aihwkit.experiments.runners.i_local.InferenceLocalRunner`   Runner for executing an inference experiments locally
```
Comment (kaoutar55) [inline docs/source/using_experiments.rst:69]: remove "an" before inference

File: docs/source/using_experiments.rst [hunk 5]
```diff
@@ -60,12 +61,14 @@ retrieving its results.
...
+:class:`~aihwkit.experiments.runners.i_cloud.InferenceCloudRunner`   Runner for executing an inference experiments in the cloud
```
Comment (kaoutar55) [inline docs/source/using_experiments.rst:70]: remove "an" before inference

**Review by kaoutar55 — COMMENTED** [review]
Thanks @kkvtran just some small typos that need to be fixed as I indicated in my comments. Thank you!

File: docs/source/using_experiments.rst (hunk 1 above)
Comment (kkvtran) [reply docs/source/using_experiments.rst:7]: done

File: docs/source/using_experiments.rst (hunk 5 above)
Comment (kkvtran) [reply docs/source/using_experiments.rst:70]: done.

File: docs/source/api/aihwkit.nn.modules.conv.rst [hunk 6]
```diff
@@ -5,4 +5,3 @@ aihwkit.nn.modules.conv module
    :members:
    :undoc-members:
    :show-inheritance:
-   :exclude-members: randn, zeros, arange, cat, ones
```
Comment (maljoras) [inline docs/source/api/aihwkit.nn.modules.conv.rst:8]: Why is this line always deleted? Is this not needed anymore?

File: docs/source/api/aihwkit.simulator.rpu_base.rst [hunk 7]
```diff
@@ -1,16 +0,0 @@
-aihwkit.simulator.rpu\_base module
-==================================
-
-.. automodule:: aihwkit.simulator.rpu_base
-   :members:
-   :undoc-members:
-   :show-inheritance:
-
```
Comment (maljoras) [inline docs/source/api/aihwkit.simulator.rpu_base.rst:8]: Is this on purpose deleted?

File: docs/source/api/aihwkit.simulator.rst [hunk 8]
```diff
@@ -23,4 +22,4 @@ Submodules
 .. toctree::
    :maxdepth: 4
 
-   aihwkit.inference
```
Comment (maljoras) [inline docs/source/api/aihwkit.simulator.rst:26]: `aihwkit.simulator.noise_models` is legacy any should be removed. `aihwkit.inference` is the correct sub module

File: docs/source/api/aihwkit.utils.rst [hunk 9]
```diff
@@ -13,4 +13,6 @@ Submodules
    :maxdepth: 4
 
    aihwkit.utils.analog_info
+   aihwkit.utils.fitting
    aihwkit.utils.visualization
+   aihwkit.utils.visualization_web
```
Comment (maljoras) [inline docs/source/api/aihwkit.utils.rst:18]: `visualization_web` not needed

File: docs/source/api/aihwkit.utils.visualization_web.rst [hunk 10]
```diff
@@ -0,0 +1,7 @@
+aihwkit.utils.visualization\_web module
+=======================================
+
```
Comment (maljoras) [inline docs/source/api/aihwkit.utils.visualization_web.rst:3]: not needed

**Review by maljoras — CHANGES_REQUESTED** [review]

File: docs/source/api/aihwkit.simulator.rpu_base.rst (hunk 7 above)
Comment (kkvtran) [reply docs/source/api/aihwkit.simulator.rpu_base.rst:8]: This is automatically generated using make apidoc.   I can take a look to see why it was removed.

File: docs/source/api/aihwkit.nn.modules.conv.rst (hunk 6 above)
Comment (kkvtran) [reply docs/source/api/aihwkit.nn.modules.conv.rst:8]: it is not needed anymore.

File: docs/source/api/aihwkit.utils.rst (hunk 9 above)
Comment (kkvtran) [reply docs/source/api/aihwkit.utils.rst:18]: I added this to the excluded path in the Makefile.

File: docs/source/api/aihwkit.simulator.rst (hunk 8 above)
Comment (kkvtran) [reply docs/source/api/aihwkit.simulator.rst:26]: removed.

**Review by maljoras — DISMISSED** [review]
Thanks @kkvtran !

**Review by maljoras — APPROVED** [review]

**Pull request merged**